src/routes/oath.ts

```diff
@@ -61,16 +61,8 @@ const getSaasquatchToken = async () =>  {
 // 1. Send user to authorization page
 router.get('/hubspot', async (req, res) => {
     if(isAuthorized(req.sessionID)) {
-        // If authorized get contacts from HubSpot API
-        const accessToken = tokenStore[req.sessionID];
-        const headers = {
-            Authorization: `Bearer ${accessToken}`,
-            'Content-Type': 'application/json'
-        };
-        const contacts = 'https://api.hubapi.com/contacts/v1/lists/all/contacts/recent';
-        try{
-            const resp = await axios.get(contacts, {headers});
-            res.status(200).send(resp.data);
+        try {
+            res.status(200).write("<script>window.opener.location = 'https://app.hubspot.com'; window.close();</script>");
```

Comment:
I tried this out and it wasn't working for me, but I did some research and I think it's because I am using Firefox and it doesn't show this immediately. According to this: https://stackoverflow.com/questions/6258210/how-can-i-output-data-before-i-end-the-response we should just be able to include in the header the content type to make this work. But it also seems like we should include a res.end() because res.write() keeps the connection open and doesn't allow other requests until it is ended. I *think* . The below works on Firefox, but I did not test Chrome as I am on Linux.
```suggestion
            res.setHeader('Connection', 'Transfer-Encoding');
            res.setHeader('Content-Type', 'text/html; charset=utf-8');
            res.setHeader('Transfer-Encoding', 'chunked');

            res.write("<script>window.opener.location = 'https://app.hubspot.com'; window.close();</script>");
            setTimeout(function() {
                res.end();
            }, 1000);
```

Reply:
I think changing the response to .send instead of .write should fix this problem but I'll double check.

Reply:
Oh yea that fixed it too for me haha, nice catch!

Reply:
Yeah I forgot I changed that when I was trying a few different things haha good thing you checked!